.github/workflows/test.yml

```diff
@@ -9,13 +9,16 @@ jobs:
       matrix:
         os: [ubuntu-24.04]
         # See doc/dev.md for GHC version support policy
-        ghc: [9.10.1, 9.8.1, 9.6.1, 9.4.5, 9.2.2, 9.0.2, 8.10.7, 8.8.4, 8.6.5]
+        ghc: [9.12.1, 9.10.1, 9.8.1, 9.6.1, 9.4.5, 9.2.2, 9.0.2, 8.10.7, 8.8.4, 8.6.5]
       fail-fast: false
-    uses: GaloisInc/.github/.github/workflows/haskell-ci.yml@v1.2
+    uses: GaloisInc/.github/.github/workflows/haskell-ci.yml@v1.4
     with:
       # See doc/dev.md for development practices around warnings.
       #
       # See Note [Parallelism] in the `haskell-ci` action for why `-j`.
-      configure-flags: --enable-tests --ghc-options='-Werror=compat -Werror=default -j'
+      #
+      # For `--allow-newer`, see
+      # https://github.com/hedgehogqa/haskell-hedgehog-classes/issues/63
```

Comment:
I wonder if we should wait until there is a resolution to https://github.com/hedgehogqa/haskell-hedgehog-classes/issues/63 before proceeding here. I'm not opposed to using `--allow-newer` in case of an emergency, but then again, https://github.com/hedgehogqa/haskell-hedgehog-classes/issues/63 was only opened a day ago.

Reply:
There has been no movement on that issue in two weeks, how would you like to proceed?

I've submitted a PR: https://github.com/hedgehogqa/haskell-hedgehog-classes/pull/64

I built and ran tests locally and they passed.

Reply:
I'll echo my usual sentiments that dependencies with uncertain maintainership make me nervous, so we should keep on eye on this going forward. That being said, the amount of hacks we need to do to work around this aren't too onerous, and they should be confined to `parameterized-utils` given that it only affects its test suite. I think we can move forward with this.